CHANGELOG.md

```diff
@@ -1,5 +1,60 @@
 # Changelog
 
+## v0.32.1
+
+*July 12, 2019*
+
+Special thanks to external contributors on this release: 
+@ParthDesai, @climber73, @jim380, @ashleyvega
+
+This release contains breaking changes to our libs folder and ABCI namely:
+- Add a "Recheck Tx" indicator 
+- Removal of `db/debugDB`, `common/colors.go`, `errors/errors.go`, `common/heap.go`, `date.go`, `io.go` and
```

Comment:
I would remove this and expand on RecheckTx indicator instead (how applications can use it with examples)